ui/apps/platform/src/Containers/ComplianceEnhanced/Dashboard/types.ts

```diff
@@ -3,3 +3,21 @@ export type PassingRateData = {
     passing: number;
     link: string;
 };
+
+interface ComplianceScanStatsShim {
+    scanName: string;
+    numberOfChecks: number;
+    numberOfFailingChecks: number;
+    numberOfPassingChecks: number;
+    lastScan: string;
```

Comment:
Based on the new API doc, we may want to add the backend types as comments
```suggestion
    numberOfChecks: number; // int32
    numberOfFailingChecks: number; // int32
    numberOfPassingChecks: number; // int32
    lastScan: string;  // ISO 8601 date string
```